tracks/csharp/exercises/space-age/mentoring.md

```diff
@@ -0,0 +1,84 @@
+### Reasonable solution
+
+- using Dictionary
+    ```csharp
```

Comment:
This entire code block is indented with 3 spaces. As this is not really necessary, could you remove the extra indentation please?

Reply:
no issue, I will remove these.